php/php.project/src/org/netbeans/modules/php/project/ui/options/PhpOptions.java

```diff
@@ -52,6 +53,7 @@
     public static final boolean DEFAULT_DEBUGGER_SHOW_CONSOLE = false;
     public static final boolean DEFAULT_DEBUGGER_STOP_AT_FIRST_LINE = true;
     public static final boolean DEFAULT_DEBUGGER_WATCHES_AND_EVAL = false;
+    public static final boolean DEFAULT_DEBUGGER_RESOLVE_BREAKPOINTS = true;
```

Comment:
This is probably fine... maybe we could check what Xdebug version is present in the latest Ubuntu LTS?

Reply:
Ubuntu 20.04 is latest LTS and it contains Xdebug 2.9.2.
https://packages.ubuntu.com/focal/php-xdebug

Reply:
Great, thanks for checking it.